test/libp2p/capability.cpp

```diff
@@ -0,0 +1,157 @@
+/*
+This file is part of cpp-ethereum.
+
+cpp-ethereum is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+cpp-ethereum is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file capability.cpp
+* @author Vladislav Gluhovsky <vlad@ethdev.com>
+* @date May 2015
+*/
+
+#include <boost/test/unit_test.hpp>
+#include <chrono>
+#include <thread>
+#include <libp2p/Common.h>
+#include <libp2p/Host.h>
+#include <libp2p/Session.h>
+#include <libp2p/Capability.h>
+#include <libp2p/HostCapability.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::p2p;
+
+struct P2PFixture
+{
+	P2PFixture() { dev::p2p::NodeIPEndpoint::test_allowLocal = true; }
+	~P2PFixture() { dev::p2p::NodeIPEndpoint::test_allowLocal = false; }
+};
+
+struct VerbosityHolder
+{
+	int m_oldLogVerbosity;
+	VerbosityHolder() : m_oldLogVerbosity(g_logVerbosity) { g_logVerbosity = 10; }
```

Comment:
No space before ":" (except in the ternary operator which is one on both sides).